### PR TITLE
Add development and source map information to README.md.

### PR DESCRIPTION
**Pull request by rpancham**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
\ No newline at end of file
+# openvino-repo-syncher
+
+This repository serves as a centralized hub for managing various OpenVINO repositories and automating the merge process between branches or repositories using GitHub Actions.
+
+## Purpose
+
+The primary purpose of this repository is to streamline the management of OpenVINO-related repositories by automating the synchronization of changes and facilitating seamless collaboration among contributors.
+
+## Actions
```
Comment (z103cb) [inline README.md]: Remove the extra space character at the end of the line.

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+
+- Auto-Merge Workflow: Automates the process of merging changes between specified repositories and branches based on predefined configurations.
+
+- Source Map (source_map.yaml) : The source map defines the relationships between source and destination repositories for automerging changes.
+
+## Workflow Configuration
+
+The workflow is triggered either manually or on a schedule. It automates the process of syncing changes between upstream and downstream repositories.
+
+## Development
```
Comment (z103cb) [inline README.md:21]: Add a blank line after this line.

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+- Attempt rebase if behind: If the downstream repository is behind the upstream repository, an attempt is made to rebase the changes onto the       downstream branch.
+- Push changes to downstream: Pushing the merged changes to the downstream repository.
+
+This automated process ensures seamless integration of changes from the upstream repository into the downstream repository, facilitating efficient collaboration and code management.
+
+- Source Map
+
+The source map defines the relationships between source and destination repositories for automerging changes. It is configured in [`source_map.yml`](source_map.yml) and includes details such as the repository names, automerge settings, synchronization modes, source URLs, and branch information.
+
+The source map enables the auto-merge workflow to synchronize changes between repositories according to the defined configurations, ensuring smooth and reliable management of code changes across different branches and repositories.
```
Comment (z103cb) [inline README.md]: Add a single trailing line at end of the file.

**Review by z103cb — COMMENTED** [review]
Small nitpiks around formatting.

You should consider adding the Markdown Linter extention for Visual Code to help out with formatting issues.

Also consider configuring your editor to always add the newline at the end of the file.

**Review by z103cb — APPROVED** [review]
LGTM

**Review by Xaenalt — APPROVED** [review]

**Pull request merged**